index.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+#container { display: flex; }
+
+nav {
+  padding-top: 10px;
+}
+
+@media only screen and (max-width: 600px) {
+  nav {
+    display: flex;
+    justify-content: space-between;
+    top: 0;
+    width: 100%;
+    position: fixed;
+  }
+
+  .fa-bars {
+    color: blue;
+    padding-right: 20px;
+  }
+
+  .fa-bars:hover {
+    opacity: 0.5;
+  }
+
+  .catch-text-sec {
+    background-image: url(/assets/headerTitle.svg);
```

Comment:
- [ ] Please 🙏🏽 make sure the background image is showing in your browser, we could correct the path by adding a dot at the beginning 👉🏽 `./assets/...`